patches/chrome-browser-themes-theme_properties.cc.patch

```diff
@@ -1,20 +0,0 @@
-diff --git a/chrome/browser/themes/theme_properties.cc b/chrome/browser/themes/theme_properties.cc
-index 733d3521fb2ecb8dd47c68cc9ca451752bdec04e..e52c5cecb77044d2b33bb2917e850da1fbd4a058 100644
---- a/chrome/browser/themes/theme_properties.cc
-+++ b/chrome/browser/themes/theme_properties.cc
-@@ -10,6 +10,7 @@
- #include "base/optional.h"
- #include "base/strings/string_split.h"
- #include "base/strings/string_util.h"
-+#include "brave/browser/themes/theme_properties.h"
- #include "build/build_config.h"
- #include "chrome/browser/themes/browser_theme_pack.h"
- #include "ui/base/material_design/material_design_controller.h"
-@@ -290,6 +291,7 @@ color_utils::HSL ThemeProperties::GetDefaultTint(int id, bool incognito) {
- 
- // static
- SkColor ThemeProperties::GetDefaultColor(int id, bool incognito) {
```

Comment:
Some places like below uses this method directly to get color.
Need to change not to use this.
```
SkColor BrowserNonClientFrameView::GetFrameColor(bool active) const {
  extensions::HostedAppBrowserController* hosted_app_controller =
      browser_view()->browser()->hosted_app_controller();
  if (hosted_app_controller && hosted_app_controller->GetThemeColor())
    return *hosted_app_controller->GetThemeColor();

  ThemeProperties::OverwritableByUserThemeProperty color_id;
  if (ShouldPaintAsSingleTabMode()) {
    color_id = ThemeProperties::COLOR_TOOLBAR;
  } else {
    color_id = active ? ThemeProperties::COLOR_FRAME
                      : ThemeProperties::COLOR_FRAME_INACTIVE;
  }
  return ShouldPaintAsThemed()
             ? GetThemeProviderForProfile()->GetColor(color_id)
             : ThemeProperties::GetDefaultColor(color_id,
                                                browser_view_->IsIncognito());
}
```